tools/wltests/platforms/hikey960_android-4.4/definitions

```diff
@@ -5,19 +5,21 @@
 
 PLATFORM_NAME="Hikey960_Android"
 
+BOOT_UEFI="${BOOT_UEFI:-YES}"
```

Comment:
As per previous comment, this should be better defined as:
```bash
BOOTLOADER=${BOOTLOADER:-uefi}
```
at the end of the following "Boot Image Configuration" section